### PR TITLE
feature：seata benchmark

### PR DESCRIPTION
**Pull request by wt-better**
Building a benchmark for seata. benchmark is divided into three parts from micro to macro:
- The first part, using jmh for micro-benchmark testing as a performance baseline, applying atomic methods
- The second part is basic benchmark testing of methods or services, mainly profiler and tracing.
- The third part, AT, tcc and other transaction modes, business scenario testing

### CODE REVIEW

File: seata-benchmark/seata-benchmark-baseline/src/main/java/org/apache/seata/benchmark/baseline/BenchmarkChartGenerator.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            File chartFile = new File(modulePath + "chart/" + className + ".png");
+            ChartUtils.saveChartAsPNG(chartFile, barChart, 640, 480);
+
+            LOGGER.info("Generated chart: " + chartFile.getAbsolutePath());
+        }
+    }
+
+    public static String readFileContent(File file) throws IOException {
+        StringBuilder contentBuilder = new StringBuilder();
+        // 使用当前线程的类加载器获取资源作为流
```
Comment (xjlgod) [inline seata-benchmark/seata-benchmark-baseline/src/main/java/org/apache/seata/benchmark/baseline/BenchmarkChartGenerator.java:100]: Use english comment here.

File: seata-benchmark/seata-benchmark-scenario/src/main/resources/benchmark.sh
Comment (xjlgod) [inline seata-benchmark/seata-benchmark-scenario/src/main/resources/benchmark.sh:1]: What is the use of this file.

File: seata-benchmark/seata-benchmark-baseline/src/main/java/org/apache/seata/benchmark/baseline/BenchmarkChartGenerator.java (hunk 1 above)
Comment (wt-better) [reply seata-benchmark/seata-benchmark-baseline/src/main/java/org/apache/seata/benchmark/baseline/BenchmarkChartGenerator.java:100]: ok, i will fix

File: seata-benchmark/seata-benchmark-scenario/src/main/resources/benchmark.sh
Comment (wt-better) [reply seata-benchmark/seata-benchmark-scenario/src/main/resources/benchmark.sh:1]: Strictly speaking, the scenario is not implemented yet, this is holder, can delete.

**Pull request closed without merging**